### PR TITLE
fix(sri-history): correct SRI for various axe files

### PR DESCRIPTION
**Pull request by WilcoFiers**
Explanation for why this went wrong is in #2042.

## Reviewer checks

**Required fields, to be filled out by PR reviewer(s)**

- [x] Follows the commit message policy, appropriate for next version
- [x] Code is reviewed for security

### CODE REVIEW

**Comment by stephenmathieson** [comment]
Why were these incorrect?

**Comment by straker** [comment]
These were auto generated from the release script, so fixing them here won't fix the problem for the next release either. We need to figure out what is wrong with the script that generates these if it's producing invalid SRIs

File: sri-history.json [hunk 1]
```diff
@@ -140,35 +140,35 @@
...
+		"axe.min.js": "sha256-z1eK+dbxVx44jv/XqW4xYJlyA9T9zXuxMlJbNJfTvOg="
 	},
 	"3.4.2": {
 		"axe.js": "sha256-1O2tV4TBC/8pYjxbIwj192fUX1SJvsjn8KQH3PdJUuU=",
 		"axe.min.js": "sha256-XXgQWSZNsMKcqEQnEU2BDkIfCzV6gCjOjrwnrzioYm4="
 	},
 	"3.5.0": {
-		"axe.js": "sha256-MPhNDpZVnZlXBeRKLHjKbiN4Lz4RA/Cb6FHqSsOQR/g=",
-		"axe.min.js": "sha256-Snr0xNyrSfcrWKoPPk/5335i7Aj5nrflncNblRZMMqM="
+		"axe.js": "sha256-w/zYk9v7UUawn15gNmZHGrh8WdzCI1bSxmRM2N7Uk4o=",
```
Comment (straker) [inline sri-history.json]: This number is different than the one I generate when on the #2042 branch

Comment (straker) [reply sri-history.json]:
```js
"3.5.0": {
		"axe.js": "sha256-+7Q5f49NA9ko9730M1oM2Tj3aXXwIvx5qaG92uETNp8=",
		"axe.min.js": "sha256-H4Py7XBch38wDECBdMX7eHsyHuTHUiz6DPbeFOoEzzs="
	}
```

Comment (WilcoFiers) [reply sri-history.json]: Running npm ci might not be correct either. I got these SRIs from https://cdnjs.com/libraries/axe-core. I've used it before, and other SRIs seem correct, so I trust it. If you want to know for sure, pull down axe.js from NPM, put it in the repo and validate that one.

Comment (straker) [reply sri-history.json]: You are correct. Have to use the exact axe-core from npm to build the same hash.

**Pull request merged**